web/src/lib/components/shared-components/combobox.svelte

```diff
@@ -116,8 +147,66 @@
     searchQuery = '';
     onSelect(selectedOption);
   };
+
+  const calculatePosition = (boundary: DOMRect | undefined) => {
+    dropdownDirection = getComboboxDirection(boundary);
+
+    if (!boundary) {
+      return undefined;
+    }
+
+    const vv = window.visualViewport;
```

Comment:
I get that `visualViewport` is a bit lengthy, but I'm not a fan of acronyms 😅

Reply:
Agreed, good call.